### DISCUSSION TITLE
"Modified" indicator on mod icon if settings are modified from default settings.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#19750] [Ideas]
Basically title, but just to explain.
~~I have an idea for ruleset mods to change their icon depending on what the user set a setting to.
One example is to display the rate of the Rate Changing mods replacing the acronym itself. If not possible, then it would be placed somewhere visible so that the user doesn't have to hover over to check the spesific settings and see it first glance.~~

Updated title to reflect to a different method.

**Comment by smoogipoo** [comment]
I don't think this will happen. It would be difficult to display anywhere else (rankings).

**Reply by peppy** [reply]
I think we discussed potentially having a "modified" light to at least let the user know there are some settings applied. I agree attempting to show individual settings is likely out of scope.

**Reply by Coppertine** [reply]
That seems like a good idea, I'll modify title and see if something like a "modified" light could be added.

**Comment by peppy** [comment]
To resurface this, I brought this up with @arflyte but haven't heard back. Curious if there's any progress on it?

**Comment by arflyte** [comment]
I still haven't got to this yet.
